namespaces.go

```diff
@@ -1,27 +1,49 @@
 package control
 
+// A struct representing an Ably namespace.
 type Namespace struct {
-	ID               string `json:"id,omitempty"`
-	Authenticated    bool   `json:"authenticated"`
-	Persisted        bool   `json:"persisted"`
-	PersistLast      bool   `json:"persistLast"`
-	PushEnabled      bool   `json:"pushEnabled"`
-	TlsOnly          bool   `json:"tlsOnly"`
-	ExposeTimeserial bool   `json:"exposeTimeserial"`
+	//The namespace or channel name that the channel rule will apply to. For example,
+	//if you specify namespace the namespace will be set to namespace and will match
```

Comment:
My understanding that this is copied from https://ably.com/docs/api/control-api#tag/namespaces/paths/~1apps~1{app_id}~1namespaces/post, but without the backticks around `namespace` when it is used as example of a value. 

My understanding is that when we publish the package, docs are rendered as markdown, so we should fix this.

Reply:
go doc comments are not markdown I'm affraid.

Reply:
Oh, I see... I am a bit worried about readability, but I think we can leave as is for now.